openfl/utilities/types.py

```diff
@@ -10,3 +10,15 @@
 
 Metric = namedtuple('Metric', ['name', 'value'])
 LocalTensor = namedtuple('LocalTensor', ['col_name', 'tensor', 'weight'])
+
+
+class Singleton:
```

Comment:
https://github.com/zcutlip/py-singleton-metaclasses/blob/master/pysingleton/singleton.py
Could you check this implementation?